src/utils/abi/ERC20Token.json

```diff
@@ -0,0 +1,222 @@
+[
```

Comment:
I don't know how to review this file so I'll leave it to you to double check it!